src/snowflake/snowpark/_internal/compiler/plan_compiler.py

```diff
@@ -68,6 +74,10 @@ def compile(self) -> Dict[PlanQueryType, List[Query]]:
         if self.should_start_query_compilation():
             # preparation for compilation
             # 1. make a copy of the original plan
+            start_time = time.time()
```

Comment:
since you are adding time now, can we add a timer for deepcopy, and cte optimization if enabled, large query breakdown if enabled, and then the total time